### ISSUE TITLE
Thonny IDE not displaying packages at all

### ISSUE THREAD

**Issue by arbbaza** [thonny/thonny#2849]
I am facing this unique problem,being a beginner to pico and Thonny IDE i am not able to understand what is wrong.When i search for any package using the manage packages option,it shows this error.I am unable to find any package or search it.the package i require is adafruit_imageload.Though i have the package downloaded from the circuitPython bundle.I am unable to add it manually ('it says pico write mode disabled' and through manage packages it does not show the avaialble packages at all.I am attaching the screen shots for reference
![Screenshot 2023-07-05 170714](https://github.com/thonny/thonny/assets/138644472/0599ca1c-4703-47de-8a5d-bcc7b8ba8769)
![Screenshot 2023-07-05 171059](https://github.com/thonny/thonny/assets/138644472/a6788d71-eaee-41ac-9e34-7e1c00c0febd)

_Originally posted by @arbbaza in https://github.com/thonny/thonny/issues/2848#issuecomment-1621590301_

**Comment by aivarannamaa** [comment]
Please see #1986 for the certificate error.

> unable to add it manually

Do you mean by uploading the files to the board via Thonny's file browser? Can you make the screenshot of this error?

**Comment by arbbaza** [comment]
Yes,using the 'Thonny file browser' I am unable to upload the package in the lib folder of Pico and also tried the 'manage packages' option in the IDE,through this method also I was not able to upload packages in Pico.

**Comment by aivarannamaa** [comment]
> Yes,using the 'Thonny file browser' I am unable to upload the package in the lib folder of Pico

Do you get an error?
